### PR TITLE
feat(connector)_: turn feature flag on

### PR DESCRIPTION
**Pull request by kounkou**
### What does the PR do

This PR turns on the feature flag for Connector

### Affected areas

- Any screen will be able to see popup  when triggered for `request account` or `sign transacation` from a DApp or equivalent

### Screenshot of functionality (including design for comparison)

Refer to : 

- [Create connection](https://github.com/status-im/status-desktop/pull/15565)
- [Sign transaction](https://github.com/status-im/status-desktop/pull/15662)

### Impact on end user

User will now be able to request connection with account and/oor sign transactions

### How to test

Please refer to above PR's for testing

### Risk 

No specific risks.

Tick **one**:
- [x] Low risk: 2 devs MUST perform testing as specified above and attach their results as comments to this PR **before** merging.
- [ ] High risk: QA team MUST perform additional testing in the specified affected areas **before** merging.

### CODE REVIEW

**Review by jrainville — APPROVED** [review]

**Comment by kounkou** [comment]
Taking the opportunity to test end to end working using [`status-browser-extension`](https://github.com/status-im/status-browser-extension/pull/18) before merging current PR.

**Comment by kounkou** [comment]
Tested and the communication is established. There are couple of errors on Browser plugin to fix. This should not block this PR merging anymore.

[Screencast from 2024-07-23 08-51-44.webm](https://github.com/user-attachments/assets/6d04befa-592a-473a-9736-f75c69bb8f0d)

**Review by stefandunca — APPROVED** [review]
🎉

**Pull request merged**
